src/sonic-config-engine/sonic-cfggen

```diff
@@ -287,7 +326,7 @@ def main():
 
     if args.from_db:
         if args.namespace is None:
-            configdb = ConfigDBConnector(**db_kwargs)
+            configdb = ConfigDBConnector(use_unix_socket_path=True, **db_kwargs)
```

Comment:
This change will require `sudo` permission for DB read access. Same issue introduced on SONiC master:
```
Aug 14 04:43:34.832529 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/bin/sonic-cfggen", line 416, in <module>
Aug 14 04:43:34.832727 cab18-5-dut INFO pmon.sh[4315]:     main()
Aug 14 04:43:34.832835 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/bin/sonic-cfggen", line 343, in main
Aug 14 04:43:34.832951 cab18-5-dut INFO pmon.sh[4315]:     configdb.connect()
Aug 14 04:43:34.833060 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/configdb.py", line 74, in connect
Aug 14 04:43:34.833151 cab18-5-dut INFO pmon.sh[4315]:     self.db_connect('CONFIG_DB', wait_for_init, retry_on)
Aug 14 04:43:34.833240 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/configdb.py", line 69, in db_connect
Aug 14 04:43:34.833350 cab18-5-dut INFO pmon.sh[4315]:     SonicV2Connector.connect(self, self.db_name, retry_on)
Aug 14 04:43:34.833444 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/dbconnector.py", line 250, in connect
Aug 14 04:43:34.833534 cab18-5-dut INFO pmon.sh[4315]:     self.dbintf.connect(db_id, retry_on)
Aug 14 04:43:34.833623 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/interface.py", line 171, in connect
Aug 14 04:43:34.833725 cab18-5-dut INFO pmon.sh[4315]:     self._onetime_connect(db_id)
Aug 14 04:43:34.833816 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/interface.py", line 183, in _onetime_connect
Aug 14 04:43:34.833906 cab18-5-dut INFO pmon.sh[4315]:     client.config_set('notify-keyspace-events', self.KEYSPACE_EVENTS)
Aug 14 04:43:34.833998 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 1243, in config_set
Aug 14 04:43:34.834126 cab18-5-dut INFO pmon.sh[4315]:     return self.execute_command('CONFIG SET', name, value)
Aug 14 04:43:34.834217 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 898, in execute_command
Aug 14 04:43:34.834307 cab18-5-dut INFO pmon.sh[4315]:     conn = self.connection or pool.get_connection(command_name, **options)
Aug 14 04:43:34.834429 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 1192, in get_connection
Aug 14 04:43:34.834733 cab18-5-dut INFO pmon.sh[4315]:     connection.connect()
Aug 14 04:43:34.834846 cab18-5-dut INFO pmon.sh[4315]:   File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 563, in connect
Aug 14 04:43:34.834961 cab18-5-dut INFO pmon.sh[4315]:     raise ConnectionError(self._error_message(e))
Aug 14 04:43:34.835068 cab18-5-dut INFO pmon.sh[4315]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
```
Please double check.

Reply:
@akokhan, that correct. Is there a foreseeable issue to using sudo with sonic-cfggen?

Reply:
It is worth noting that this is inline with multi-asic platform were it uses `/var/run/redis/redis.sock` already.

Reply:
The issues like this can be seen in syslog in latest SONiC master images. So, it's not just about running sonic-cfggen with sudo. Looks like if we do this change, some additional changes related to pmon.sh/bgp.sh/teamd.sh/... are required.

Reply:
@akokhan I used multiple images from master and did not see the above error. Can you please let me know which hash was the image based upon and exact steps that leads to the above error?

Reply:
I see this issue on all latest SONiC master on Barefoot platform (e.g., SONiC.HEAD.740-dirty-20200813.174118). But I'm pretty sure the issue is common. To reproduce the issue, just deploy SONiC image and run the command as follows:
```
admin@cab18-1-dut:~$ sudo cat /var/log/syslog | grep /var/run/redis/redis.sock
Aug 19 14:59:48.224809 cab18-1-dut INFO database#supervisord: redis 21:M 19 Aug 2020 14:59:48.224 * The server is now ready to accept connections at /var/run/redis/redis.sock
Aug 19 14:59:49.908134 cab18-1-dut INFO pmon.sh[1050]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:50.214897 cab18-1-dut INFO bgp.sh[1045]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:53.362164 cab18-1-dut INFO teamd.sh[1825]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:54.910194 cab18-1-dut INFO lldp.sh[2095]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:54.956994 cab18-1-dut INFO mgmt-framework.sh[2094]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:54.962366 cab18-1-dut INFO dhcp_relay.sh[2097]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:55.264211 cab18-1-dut INFO radv.sh[2093]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 19 14:59:58.186250 cab18-1-dut INFO telemetry.sh[2950]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
```

Reply:
@akokhan can you please open a separate issue as I do not think a fix, if any, would be part of this PR.

I just a did quick test again with an image I built and I do not see the issue. It is not clear to me why this issue would be platform related rather than image related. SONiC does all configuration in an elevated access mode and so rw access to redis.sock shouldn't be an issue. Please attach show tech to the issue when opened.
```
admin@str-s6000-acs-14:~$ sudo cat /var/log/syslog | grep /var/run/redis/redis.sock
admin@str-s6000-acs-14:~$ show ver

SONiC Software Version: SONiC.taahme_fix-delay-start-of-snmp-telemetry.0-dirty-20200818.215531
Distribution: Debian 10.5
Kernel: 4.19.0-9-2-amd64
admin@str-s6000-acs-14:~$ sudo zgrep redis /var/log/syslog*
/var/log/syslog.1:Aug 19 16:35:34 sonic systemd[1]: var-lib-docker-overlay2-8f170ecf842fc184be5a16630d3e6144ef46e9e2659795fcfd1a26ebee9f86ee-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:35:34 sonic systemd[1]: var-lib-docker-overlay2-8f170ecf842fc184be5a16630d3e6144ef46e9e2659795fcfd1a26ebee9f86ee-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:35:35 sonic systemd[1]: var-lib-docker-overlay2-8f170ecf842fc184be5a16630d3e6144ef46e9e2659795fcfd1a26ebee9f86ee-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:35:35 sonic systemd[1]: var-lib-docker-overlay2-8f170ecf842fc184be5a16630d3e6144ef46e9e2659795fcfd1a26ebee9f86ee-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:36:21.223668 str-s6000-acs-14 INFO systemd[1]: var-lib-docker-overlay2-f544adc1459959cc3694473b6d8288547e7ca07459a49ee93becd4b6dbda1987-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:36:23.074462 str-s6000-acs-14 INFO systemd[1]: var-lib-docker-overlay2-f544adc1459959cc3694473b6d8288547e7ca07459a49ee93becd4b6dbda1987-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:36:25.084659 str-s6000-acs-14 INFO systemd[1]: var-lib-docker-overlay2-f544adc1459959cc3694473b6d8288547e7ca07459a49ee93becd4b6dbda1987-merged-run-redis.mount: Succeeded.
/var/log/syslog.1:Aug 19 16:36:54.120155 str-s6000-acs-14 NOTICE swss#orchagent: :- startRecording: started recording: sairedis.rec
/var/log/syslog.1:Aug 19 16:36:54.123551 str-s6000-acs-14 NOTICE swss#orchagent: :- initSaiRedis: Enable redis pipeline
/var/log/syslog.1:Aug 19 16:36:57.267511 str-s6000-acs-14 NOTICE swss#orchagent: :- sai_redis_notify_syncd: switched ASIC to INIT VIEW
/var/log/syslog.1:Aug 19 16:36:57.271420 str-s6000-acs-14 NOTICE swss#orchagent: :- sai_redis_notify_syncd: clearing current local state since init view is called on initialized switch
/var/log/syslog.1:Aug 19 16:37:12.381496 str-s6000-acs-14 NOTICE syncd#syncd: :- helperSaveDiscoveredObjectsToRedis: putting ALL discovered objects to redis
/var/log/syslog.1:Aug 19 16:37:13.724556 str-s6000-acs-14 NOTICE syncd#syncd: :- helperSaveDiscoveredObjectsToRedis: save discovered objects to redis took 1.343632 sec
/var/log/syslog.1:Aug 19 16:37:14.569741 str-s6000-acs-14 NOTICE swss#orchagent: :- sai_redis_notify_syncd: switched ASIC to APPLY VIEW
admin@str-s6000-acs-14:~$ sudo zgrep sonic-cfggen /var/log/syslog*
/var/log/syslog.1:Aug 19 16:35:45 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen -H -m -j /etc/sonic/init_cfg.json   --write-to-db
/var/log/syslog.1:Aug 19 16:35:45 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen -H -m -j /etc/sonic/init_cfg.json   --write-to-db
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -d -t /usr/share/sonic/device/x86_64-dell_s6000_s1220-r0/Force10-S6000/buffers.json.j2 > /tmp/buffers.json
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -d -t /usr/share/sonic/device/x86_64-dell_s6000_s1220-r0/Force10-S6000/qos.json.j2 -y /etc/sonic/sonic_version.yml > /tmp/qos.json
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -j /tmp/buffers.json --write-to-db
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -j /tmp/qos.json --write-to-db
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -d -t /usr/share/sonic/device/x86_64-dell_s6000_s1220-r0/Force10-S6000/buffers.json.j2 > /tmp/buffers.json
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -d -t /usr/share/sonic/device/x86_64-dell_s6000_s1220-r0/Force10-S6000/qos.json.j2 -y /etc/sonic/sonic_version.yml > /tmp/qos.json
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -j /tmp/buffers.json --write-to-db
/var/log/syslog.1:Aug 19 16:35:54 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen  -j /tmp/qos.json --write-to-db
/var/log/syslog.1:Aug 19 16:35:56 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen -d --print-data > /etc/sonic/config_db.json
/var/log/syslog.1:Aug 19 16:35:56 sonic config-setup[1036]: Running command: /usr/local/bin/sonic-cfggen -d --print-data > /etc/sonic/config_db.json
admin@str-s6000-acs-14:~$ 

```

Reply:
@akokhan I noticed from version string post that you are building your own image. can you please do a clean build and see if the issue happens? You can alternatively use an image from Jenkins build artifacts.

Reply:
@tahmed-dev , I did not build my own image. It was taken from Azure Jenkins 
https://sonic-jenkins.westus2.cloudapp.azure.com/job/barefoot/job/buildimage-bf-all/740/

Reply:
One more set of simple steps to reproduce the issue:
```
sudo systemctl restart bgp

admin@sonic:~$ sudo cat /var/log/syslog | grep -B 30 -A 5 /var/run/redis/redis.sock
Aug 20 10:55:05.963130 sonic NOTICE admin: Stopped bgp service...
Aug 20 10:55:05.967556 sonic INFO systemd[1]: bgp.service: Succeeded.
Aug 20 10:55:05.969013 sonic INFO systemd[1]: Stopped BGP container.
Aug 20 10:55:05.972528 sonic INFO systemd[1]: Starting BGP container...
Aug 20 10:55:05.979468 sonic NOTICE admin: Starting bgp service...
Aug 20 10:55:06.231106 sonic NOTICE admin: Warm boot flag: bgp false.
Aug 20 10:55:06.238290 sonic NOTICE admin: Fast boot flag: bgp .
Aug 20 10:55:06.734560 sonic INFO bgp.sh[28088]: Traceback (most recent call last):
Aug 20 10:55:06.734827 sonic INFO bgp.sh[28088]:   File "/usr/local/bin/sonic-cfggen", line 416, in <module>
Aug 20 10:55:06.735145 sonic INFO bgp.sh[28088]:     main()
Aug 20 10:55:06.735326 sonic INFO bgp.sh[28088]:   File "/usr/local/bin/sonic-cfggen", line 343, in main
Aug 20 10:55:06.735498 sonic INFO bgp.sh[28088]:     configdb.connect()
Aug 20 10:55:06.735671 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/configdb.py", line 74, in connect
Aug 20 10:55:06.735854 sonic INFO bgp.sh[28088]:     self.db_connect('CONFIG_DB', wait_for_init, retry_on)
Aug 20 10:55:06.736030 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/configdb.py", line 69, in db_connect
Aug 20 10:55:06.736202 sonic INFO bgp.sh[28088]:     SonicV2Connector.connect(self, self.db_name, retry_on)
Aug 20 10:55:06.736377 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/dbconnector.py", line 250, in connect
Aug 20 10:55:06.736549 sonic INFO bgp.sh[28088]:     self.dbintf.connect(db_id, retry_on)
Aug 20 10:55:06.736778 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/interface.py", line 171, in connect
Aug 20 10:55:06.736941 sonic INFO bgp.sh[28088]:     self._onetime_connect(db_id)
Aug 20 10:55:06.737094 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/swsssdk/interface.py", line 183, in _onetime_connect
Aug 20 10:55:06.737249 sonic INFO bgp.sh[28088]:     client.config_set('notify-keyspace-events', self.KEYSPACE_EVENTS)
Aug 20 10:55:06.737408 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 1243, in config_set
Aug 20 10:55:06.737562 sonic INFO bgp.sh[28088]:     return self.execute_command('CONFIG SET', name, value)
Aug 20 10:55:06.737714 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 898, in execute_command
Aug 20 10:55:06.737872 sonic INFO bgp.sh[28088]:     conn = self.connection or pool.get_connection(command_name, **options)
Aug 20 10:55:06.738027 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 1192, in get_connection
Aug 20 10:55:06.738180 sonic INFO bgp.sh[28088]:     connection.connect()
Aug 20 10:55:06.738334 sonic INFO bgp.sh[28088]:   File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 563, in connect
Aug 20 10:55:06.738487 sonic INFO bgp.sh[28088]:     raise ConnectionError(self._error_message(e))
Aug 20 10:55:06.738642 sonic INFO bgp.sh[28088]: redis.exceptions.ConnectionError: Error 13 connecting to unix socket: /var/run/redis/redis.sock. Permission denied.
Aug 20 10:55:06.836439 sonic INFO bgp.sh[28088]: Removing obsolete bgp container with HWSKU montara
Aug 20 10:55:06.902063 sonic INFO bgp.sh[28088]: bgp
Aug 20 10:55:06.905879 sonic INFO bgp.sh[28088]: Creating new bgp container with HWSKU
Aug 20 10:55:07.085741 sonic INFO bgp.sh[28088]: a9f9487d078ec6ae0d3ca4c793bcfa41975d0c50a7decf48331f50a162d0e438
Aug 20 10:55:07.203221 sonic INFO containerd[505]: time="2020-08-20T10:55:07.202160280Z" level=info msg="shim containerd-shim started" address="/containerd-shim/moby/a9f9487d078ec6ae0d3ca4c793bcfa41975d0c50a7decf48331f50a162d0e438/shim.sock" debug=false pid=28176
admin@sonic:~$ 
```

The issue flow is:
```
bgp.service ->  /usr/local/bin/bgp.sh ->  /usr/bin/bgp.sh
```

Probably because `User=admin` in `bgp.service`:
```
[Service]
User=admin
ExecStartPre=/usr/local/bin/bgp.sh start
```
As far as I see, you are using some custom image. Could you please check `User=admin` in `bgp.service in your image.
Thank you.

Reply:
@akokhan Let's move this duscussion to a separate issue. The issue will be better tracked with a separate issue item. I'll resolve this one here.

I just did those steps described about and the issue is not reproducible.

```
admin@str-s6000-acs-14:~$ docker ps -a
CONTAINER ID        IMAGE                                COMMAND                  CREATED             STATUS                     PORTS               NAMES
3f3b4a351f67        docker-sonic-telemetry:latest        "/usr/bin/supervisord"   12 hours ago        Exited (0) 6 seconds ago                       telemetry
a6f88044098a        docker-snmp:latest                   "/usr/bin/supervisord"   12 hours ago        Up About a minute                              snmp
20f662ed09ba        docker-router-advertiser:latest      "/usr/bin/docker-ini…"   12 hours ago        Up 3 minutes                                   radv
17656a3d4423        docker-lldp:latest                   "/usr/bin/docker-lld…"   12 hours ago        Up 3 minutes                                   lldp
76a42632749b        docker-sonic-mgmt-framework:latest   "/usr/bin/supervisord"   12 hours ago        Up 3 minutes                                   mgmt-framework
5e520cb815de        docker-dhcp-relay:latest             "/usr/bin/docker_ini…"   12 hours ago        Up 3 minutes                                   dhcp_relay
c0f0c5218364        docker-syncd-brcm:latest             "/usr/bin/supervisord"   12 hours ago        Up 3 minutes                                   syncd
1d6242ec3a5e        docker-teamd:latest                  "/usr/bin/supervisord"   12 hours ago        Up 3 minutes                                   teamd
994fb765a8de        docker-orchagent:latest              "/usr/bin/docker-ini…"   12 hours ago        Up 4 minutes                                   swss
9d6996e1a526        docker-fpm-frr:latest                "/usr/bin/docker_ini…"   12 hours ago        Up 4 minutes                                   bgp
a01a6dd60013        docker-platform-monitor:latest       "/usr/bin/docker_ini…"   12 hours ago        Up 4 minutes                                   pmon
0e7a7716ebe0        docker-database:latest               "/usr/local/bin/dock…"   12 hours ago        Up 4 minutes                                   database
admin@str-s6000-acs-14:~$ sudo zgrep -e 'sonic-cfggen\|redis.sock' /var/log/syslog*
Aug 20 14:10:22.010869 str-s6000-acs-14 INFO database#supervisord: redis 21:M 20 Aug 2020 14:10:22.009 * The server is now ready to accept connections at /var/run/redis/redis.sock
admin@str-s6000-acs-14:~$ show ip bgp sum

IPv4 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 8603
RIB entries 12807, using 2301 KiB of memory
Peers 4, using 82 KiB of memory
Peer groups 6, using 384 bytes of memory

Neighbor        V         AS MsgRcvd MsgSent   TblVer  InQ OutQ  Up/Down State/PfxRcd	NeighborName
10.0.0.57       4      64600    3276    3282        0    0    0 00:03:35         6400	ARISTA01T1
10.0.0.59       4      64600    3276    3280        0    0    0 00:03:35         6400	ARISTA02T1
10.0.0.61       4      64600    3276    3282        0    0    0 00:03:35         6400	ARISTA03T1
10.0.0.63       4      64600    3276    3282        0    0    0 00:03:35         6400	ARISTA04T1

Total number of neighbors 4
admin@str-s6000-acs-14:~$ sudo systemctl restart bgp
admin@str-s6000-acs-14:~$ sudo zgrep -e 'sonic-cfggen\|redis.sock' /var/log/syslog*
Aug 20 14:10:22.010869 str-s6000-acs-14 INFO database#supervisord: redis 21:M 20 Aug 2020 14:10:22.009 * The server is now ready to accept connections at /var/run/redis/redis.sock
admin@str-s6000-acs-14:~$ show ip bgp sum

IPv4 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 12804
RIB entries 12807, using 2301 KiB of memory
Peers 4, using 82 KiB of memory
Peer groups 6, using 384 bytes of memory

Neighbor        V         AS MsgRcvd MsgSent   TblVer  InQ OutQ  Up/Down State/PfxRcd	NeighborName
10.0.0.57       4      64600    3222    3225        0    0    0 00:00:54         6400	ARISTA01T1
10.0.0.59       4      64600    3221    3223        0    0    0 00:00:50         6400	ARISTA02T1
10.0.0.61       4      64600    3228    9635        0    0    0 00:01:13         6400	ARISTA03T1
10.0.0.63       4      64600    3228    7942        0    0    0 00:01:09         6400	ARISTA04T1

Total number of neighbors 4
admin@str-s6000-acs-14:~$ sudo zgrep -e 'sonic-cfggen\|redis.sock' /var/log/syslog*
Aug 20 14:10:22.010869 str-s6000-acs-14 INFO database#supervisord: redis 21:M 20 Aug 2020 14:10:22.009 * The server is now ready to accept connections at /var/run/redis/redis.sock
￼admin@str-s6000-acs-14:~$ cat /usr/lib/systemd/system/bgp.service
[Unit]
Description=BGP container
Requires=database.service
After=database.service
Requires=updategraph.service
After=updategraph.service
Before=ntp-config.service
StartLimitIntervalSec=1200
StartLimitBurst=3

[Service]
User=admin
ExecStartPre=/usr/local/bin/bgp.sh start
ExecStart=/usr/local/bin/bgp.sh wait
ExecStop=/usr/local/bin/bgp.sh stop

Restart=always
RestartSec=30

[Install]
WantedBy=multi-user.target

admin@str-s6000-acs-14:~$ show ver

SONiC Software Version: SONiC.taahme_fix-delay-start-of-snmp-telemetry.0-26e556475
Distribution: Debian 10.5
Kernel: 4.19.0-9-2-amd64
Build commit: 26e556475
Build date: Thu Aug 20 01:18:23 UTC 2020
Built by: taahme@taahme-vm0

```